### PR TITLE
Automate Psychic feats spell alterations

### PR DESCRIPTION
**Pull request by pedrogrullada**

### CODE REVIEW

File: static/lang/re-en.json [hunk 1]
```diff
@@ -2826,6 +2826,18 @@
                 "WeaponOrUnarmed": "Select a weapon or unarmed strike."
             },
             "Psychic": {
+                "PsychicRapport": {
+                    "Description": "After you @UUID[Compendium.pf2e.actionspf2e.Item.aBQ8ajvEBByv45yz]{Cast a Spell} with the mental trait, until the start of your next turn, you gain a +1 circumstance bonus to Deception, Diplomacy, and Intimidation checks, and to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.1xRFPTFtWtGJ9ELw]{Sense Motive}."
```
Comment (simonward) [inline static/lang/re-en.json:2830]:
```suggestion
                    "Description": "After you Cast a Spell with the mental trait, until the start of your next turn, you gain a +1 circumstance bonus to Deception, Diplomacy, and Intimidation checks, and to Perception checks to @UUID[Compendium.pf2e.actionspf2e.Item.1xRFPTFtWtGJ9ELw]{Sense Motive}."
```

Comment (pedrogrullada) [reply static/lang/re-en.json:2830]: Any reason it's preferred to not link the action here? Either way, I've done it inconsistently here so I have to commit to one

Comment (simonward) [reply static/lang/re-en.json:2830]: I think it's considered a common action, so typically it isn't linked. It also seems unlikely that the Cast a Spell link will be used in the context of a roll note, so it just makes it cluttered.

Comment (pedrogrullada) [reply static/lang/re-en.json:2830]: Fair enough, I'll go without links for now just to fix my inconsistency. If we want them back Tikael can let me know.

File: packs/feats/deep-roots.json [hunk 2]
```diff
@@ -24,7 +24,28 @@
...
+                "predicate": [
+                    "item:tag:deep-roots"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Psychic.DeepRoots.Description"
+                    }
+                ]
+            }
```
Comment (TikaelSol) [inline packs/feats/deep-roots.json:47]:
```suggestion
                "key": "RollOption",
                "option": "imposes:controlled",
                "placement": "spellcasting",
                "toggleable": true
            },
            {
                "itemType": "spell",
                "key": "ItemAlteration",
                "mode": "add",
                "predicate": [
                    "imposes:controlled"
                ],
                "property": "description",
                "value": [
                    {
                        "text": "PF2E.SpecificRule.Psychic.DeepRoots.Description"
                    }
                ]
            }
```
For the other one we can auto tag the spells grants with the ancestral spell tag. This this one we don't need tags, and instead go to the normal way we mark these kind of roll options, though it's likely going to be a while before it's truly automated.

File: static/lang/re-en.json [hunk 3]
```diff
@@ -2826,6 +2826,18 @@
...
+                },
+                "StrainMind": {
+                    "Description": "You add an amp that costs 1 Focus Point to the spell. Instead of paying the normal Focus Point cost, you lose Hit Points equal to four times the spell rank of the amped psi cantrip as you bleed from your nose or suffer some other visible sign of strain."
+                },
+                "ThoughtformSummoning": {
+                    "Description": "Whenever you summon a creature, you can choose to instead create a psychic construct of that creature. This grants it resistance to physical damage equal to half its level, weakness 5 to force and mental damage, and the ability to pass through enemies' spaces without needing to @UUID[Compendium.pf2e.actionspf2e.Item.21WIfSu7Xd7uKqV8]{Tumble Through} (though it can't end its turn in an occupied space)."
+                },
```
Comment (TikaelSol) [inline static/lang/re-en.json:2840]: Let's move these all down one level, under SpellModification, that's where the non amp cantrip modifications are going.

File: packs/feats/thoughtform-summoning.json [hunk 4]
```diff
@@ -24,7 +24,32 @@
...
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Psychic.ThoughtformSummoning.Description"
+                    },
+                    {
+                        "text": "@UUID[Compendium.pf2e.feat-effects.Item.Effect: Thoughtform Summoning]"
+                    }
+                ]
+            }
+        ],
```
Comment (TikaelSol) [inline packs/feats/thoughtform-summoning.json:52]:
```suggestion
        "rules": [
            {
                "itemType": "spell",
                "key": "ItemAlteration",
                "mode": "add",
                "predicate": [
                    "item:trait:summon"
                ],
                "property": "description",
                "value": [
                    {
                        "text": "PF2E.SpecificRule.Psychic.ThoughtformSummoning.Description"
                    },
                    {
                        "text": "@UUID[Compendium.pf2e.feat-effects.Item.Effect: Thoughtform Summoning]"
                    }
                ]
            }
        ],
```
Fine to not have a toggle, the effect linked will be the thing that does the optional bit if they want to use it.

**Pull request merged**